components/dashboard/sections/contributions/Contributions.tsx

```diff
@@ -341,57 +361,68 @@ const getColumns = ({ tab, intl, isIncoming, includeHostedAccounts, onlyExpected
               <div>
                 <Avatar size={32} collective={fromAccount} displayTitle={false} />
               </div>
-              <div className="overflow-hidden text-ellipsis whitespace-nowrap text-sm leading-5">
-                {fromAccount.name || fromAccount.slug}
+              <div className="overflow-hidden">
+                <div className="overflow-hidden text-ellipsis whitespace-nowrap text-sm leading-5">
+                  {fromAccount.name || fromAccount.slug}
+                </div>
+                {tier && (
+                  <div className="overflow-hidden text-ellipsis whitespace-nowrap text-xs font-normal leading-4 text-slate-700">
+                    {tier.name}
+                  </div>
+                )}
               </div>
             </div>
           }
         />
       );
     },
   };
+
   const paymentMethod = {
     accessorKey: 'paymentMethod',
     header: intl.formatMessage({ id: 'paymentmethod.label', defaultMessage: 'Payment Method' }),
     cell: ({ cell, row }) => {
       const pm = cell.getValue();
-      if (pm) {
-        return (
-          <div className="flex items-center gap-2 truncate">
-            <PaymentMethodTypeWithIcon iconSize={18} type={pm.type} />
-          </div>
-        );
-      } else if (row.original?.pendingContributionData?.paymentMethod) {
+      if (row.original?.pendingContributionData?.paymentMethod) {
         return i18nPaymentMethodProviderType(intl, row.original?.pendingContributionData?.paymentMethod);
       }
+
+      return <PaymentMethodTypeLabel type={pm?.type} />;
     },
   };
-  const status = {
-    accessorKey: 'status',
-    header: intl.formatMessage({ id: 'order.status', defaultMessage: 'Status' }),
+
+  const amount = {
+    accessorKey: 'totalAmount',
+    header: intl.formatMessage({ defaultMessage: 'Amount', id: 'Fields.amount' }),
+    meta: { className: 'text-end pr-1' },
     cell: ({ cell }) => {
-      const status = cell.getValue();
+      const amount = cell.getValue();
+      return <FormattedMoneyAmount amount={amount.valueInCents} currency={amount.currency} showCurrencyCode={false} />;
+    },
+  };
+
+  const frequency = {
+    accessorKey: 'frequency',
+    header: null,
+    meta: { className: 'text-start pl-0' },
+    cell: ({ cell }) => {
+      const frequency = cell.getValue();
       return (
-        <div data-cy="contribution-status">
-          <OrderStatusTag status={status} />
-        </div>
+        ['MONTHLY', 'YEARLY'].includes(frequency) && (
+          <span className="text-xs font-light text-slate-700">{i18nFrequency(intl, frequency)}</span>
```

Comment:
Just a suggestion, but I think this will make it seem more like a singular label together with the amount:
```suggestion
          <span className="text-sm font-light lowercase text-muted-foreground">{i18nFrequency(intl, frequency)}</span>
```

<img width="262" alt="image" src="https://github.com/user-attachments/assets/e5390ca9-e151-444e-8c4a-c4ef45c4c2a0" />